examples/remote_procedure/template/run_task.py

```diff
@@ -112,7 +112,7 @@ def handle_with_model(
     build_and_return_custom_bundle(task_dir)
     operator = Operator(db)
 
-    operator.validate_and_run_config(cfg.mephisto, shared_state)
+    operator.launch_task_run(cfg.mephisto, shared_state)
     operator.wait_for_runs_then_shutdown(skip_input=True, log_rate=30)
 
 
```

Comment:
should we update this file to use `@run_script` as well?

Reply:
Ah I did, but dropped the changes on merge accidentally (create/create). Will redo!